src/SignalR/server/Core/src/Internal/DefaultHubDispatcher.cs

```diff
@@ -381,6 +390,17 @@ async Task ExecuteInvocation()
             }
         }
 
+        private static async IAsyncEnumerable<T> ConvertStream<T>(ChannelReader<T> reader)
```

Comment:
This is built into ChannelReader. It’s ReadAllAsync

Reply:
Forgot about that API